packages/webpack/test/fixtures/main/simple.js

```diff
@@ -0,0 +1,8 @@
+function start(arr) {
+  arr.push(111)
+  return arr
+}
+
+if (start(new Array())[0] !== 111) {
```

Comment:
Is this enough to check if scuttling did not disrupt the compartment globals? From the current implementation I thought it should :sweat_smile: